app/Device.php

```diff
@@ -262,4 +257,41 @@ public function data()
     {
         return $this->hasManyThrough('App\SensorData', 'App\Sensor');
     }
+    
+    /**
+     * Is the device ready for a command
+     *
+     * @return boolean
+     */
+    public function isReadyForCommand()
+    {
+        $isReady = false;
+        
+        if ($this->cover_status == 'open' || $this->cover_status == 'closed' || $this->cover_status == 'locked')
+            $isReady = true;
+        
+        return $isReady;
```

Comment:
This can just be:
return ($this->cover_status == 'open' || $this->cover_status == 'closed' || $this->cover_status == 'locked');